method_workflows/README.md

```diff
@@ -0,0 +1,214 @@
+If a method has distinct run modes other than those concerning the three benchmarking events, the calls to those should also be parameterized. If those run modes could significantly alter the behaviour of the method, please discuss with the APAeval community whether the distinct run modes should actually be treated as distinct participants in APAeval (see [section on parameters](#parameters)). That could for example be the case if the method can be run with either mathematical model A or model B, and the expected results would be quite different. At the moment we can't foresee all possibilities, so we count on you to report and discuss any such cases. In any case, please do document extensively how the method can be used and how you employed it. 
+In general, all relevant participant parameters should be configurable in the workflow config files. Parameters, file names, run modes, etc. MUST NOT be hardcoded within the workflow. 
+
+> IMPORTANT: Do not download any other annotation files because the docs of your participant say so. Instead, create all files the participant needs from the ones provided by APAeval. If you don't know how, please don't hesitate to start discussions within the APAeval community! Chances are high that somebody already encountered a similar problem and will be able to help.
+
+3. **Post-processing:** To ensure compatibility with the OEB benchmarking events, [specifications for file formats][spec-doc] (output of method workflows = input for benchmarking workflows) are provided by APAeval. There is one specification per metric (=statistical parameter to assess performance of a participant), but calculation of several metrics can require a common input file format (thus, the file has to be created only once by the method workflow). The required method workflow outputs are a bed file containing coordinates of identified PAS (and their respective expression in tpm, if applicable) a tsv file containing information on differential expression (if applicable), and a json file containing information about compute resource and time requirements (see [output specifications][spec-doc] for detailed description of the file formats). These files have to be created within the method workflows as post-processing steps.
+
+
+_Method workflows_ should be implemented in either [Nexflow][nf] or
+[Snakemake][snakemake], and individual steps should be isolated through the
+use of either [Conda][conda] virtual environments or
```

Comment:
remove part suggesting we allow use of conda environments? Because in 'containers' section we mention we require Docker containers (and I know for a fact we've required that throughout our work)

Reply:
Ahh, I suppose containers can be created from conda env files via Biocontainers? Seems to be mentioned in 'Containers' section. Regardless, maybe best to remove the mention of conda envs here for clarity